tasks/read-client-files.yml

```diff
@@ -5,12 +5,21 @@
   register: openvpn_read_ca_file_results
   changed_when: false
 
+- name: Read TLS-auth key
+  slurp:
+    src: "{{ openvpn_keydir }}/{{ openvpn_tls_key }}"
+  no_log: true
+  register: openvpn_read_tlsauth_file_results
+  changed_when: false
+  when: openvpn_tls_auth
+
 - name: Set CA file contents as fact.
   set_fact:
     openvpn_ca_file_contents: "{{ openvpn_read_ca_file_results.stdout }}"
+    openvpn_tls_auth_file_contents: "{{ openvpn_read_tlsauth_file_results['content'] | b64decode | default('') }}"
 
 - name: Read client cert files.
-  command: cat {{ openvpn_etcdir }}/keys/{{ item }}.crt
+  command: sed -n '/-----BEGIN CERTIFICATE-----/,/-----END CERTIFICATE-----/p' {{ openvpn_keydir }}/{{ item }}.crt
```

Comment:
Not sure about motivation here. From `man openvpn`: 

```
INLINE FILE SUPPORT
       OpenVPN allows including files in the main configuration for the --ca, --cert,
       --dh,    --extra-certs,    --key,    --pkcs12,     --secret,     --crl-verify,
       --http-proxy-user-pass, --tls-auth and --tls-crypt options.

       Each inline file started by the line <option> and ended by the line </option>

       Here is an example of an inline file usage

           <cert>
           -----BEGIN CERTIFICATE-----
           [...]
           -----END CERTIFICATE-----
           </cert>

       When  using  the  inline  file feature with --pkcs12 the inline file has to be
       base64 encoded. Encoding of a .p12 file into base64 can be  done  for  example
       with OpenSSL by running openssl base64 -in input.p12
```

so I'm inclined to keep it simple and preserve the cert boundary tags.